uPortal-webapp/src/main/webapp/media/skins/respondr/defaultSkin/skin.xml

```diff
@@ -39,8 +39,8 @@
   <css included="aggregated">/ResourceServingWebapp/rs/jqueryui/1.10.3/theme/smoothness/jquery-ui-1.10.3-smoothness.min.css</css>
 
   <!-- jsTree (entity-selector) -->
-  <css included="plain" resource="true">/scripts/jstree/style.css</css>
-  <css included="aggregated" resource="true">/scripts/jstree/style.min.css</css>
+  <css included="plain" resource="true">/webjars/jstree/dist/themes/default/style.css</css>
+  <css included="aggregated" resource="true">/webjars/jstree/dist/themes/default/style.min.css</css>
```

Comment:
Hold up!

Now that I see it, I'm wondering if these lines should be moved to `common_skin.xml`.

Reply:
moved :white_check_mark: